packages/cli/src/executionLifecycleHooks/restoreBinaryDataId.ts

```diff
@@ -20,7 +20,7 @@ export async function restoreBinaryDataId(run: IRun, executionId: string) {
 	const { runData } = run.data.resultData;
 
 	const promises = Object.keys(runData).map(async (nodeName) => {
-		const binaryDataId = runData[nodeName]?.[0]?.data?.main?.[0]?.[0]?.binary?.data.id;
+		const binaryDataId = runData[nodeName]?.[0]?.data?.main?.[0]?.[0]?.binary?.data?.id;
```

Comment:
can we add a test here for this bug?

Reply:
@krynble https://github.com/n8n-io/n8n/pull/7420

Reply:
Thanks for the note @mutdmour - Ivan did add. Would you like to review?